### PR TITLE
Code Formatting

### PR DESCRIPTION
**Pull request by kensipe**
code formatting that aligns with our standard and fixes that the style checker will complain about.
this is based on existing structure and based off current master.

### CODE REVIEW

**Comment by elingg** [comment]
@kensipe, question for you, don't you need to update the Java Formatter referenced in the pom.xml accordingly?

**Comment by kensipe** [comment]
rebased to master after gradle merge

**Comment by kensipe** [comment]
@elingg the maven formatter now matches our standard.  It is also the default format (with our standard) of the IDEA code formatter.  ready to merge as soon as Travis says so.

**Comment by elingg** [comment]
hey @kensipe, I'll take a look soon. Travis is now failing, can you look into that? @adam-mesos can optionally review as well.

**Comment by kensipe** [comment]
@elingg the failure was on a maven central download... I'll double check, but this seems like a networking issue.  especially since it passed one of the CI builds.

**Comment by elingg** [comment]
Ok, I'm rerunning the build now.

**Comment by elingg** [comment]
LGTM! Any final objections before merging, @adam-mesos?

File: src/main/java/org/apache/mesos/hdfs/state/AcquisitionPhase.java [hunk 1]
```diff
@@ -6,5 +6,5 @@
   START_NAME_NODES, // Launches the both namenodes
   FORMAT_NAME_NODES, // Formats both namenodes (first with initialize, second with bootstrap
   DATA_NODES // If everything is healthy the scheduler stays here and tries to launch
-             // datanodes on any slave that doesn't have an hdfs task running on it
+  // datanodes on any slave that doesn't have an hdfs task running on it
```
Comment (adam-mesos) [inline src/main/java/org/apache/mesos/hdfs/state/AcquisitionPhase.java]: This formatting change looks a little awkward to me, but I don't care too much. It's certainly a special case, and the benefits of global formatting rules outweigh the need for special multi-line end-of-line comments.

File: src/main/java/org/apache/mesos/hdfs/state/LiveState.java [hunk 2]
```diff
@@ -73,24 +70,24 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
           || (currentAcquisitionPhase.equals(AcquisitionPhase.RECONCILING_TASKS)
-              && !isNameNode1Initialized())
+          && !isNameNode1Initialized())
           || (status.getMessage().equals(HDFSConstants.NAME_NODE_BOOTSTRAP_MESSAGE)
-              && !isNameNode1Initialized())) {
+          && !isNameNode1Initialized())) {
```
Comment (adam-mesos) [inline src/main/java/org/apache/mesos/hdfs/state/LiveState.java]: These look wrong to me. The &&'s are nested within the previous line's (), so I think it reads better for them to be indented to match.

**Comment by adam-mesos** [comment]
A couple of minor nits, but I won't hold back the merge for them. @kensipe might have better ideas of how to address my concerns.

**Comment by kensipe** [comment]
oh... formatting... oh how I loathe and welcome you.  I'll see what we can do and be back.

**Comment by kensipe** [comment]
@elingg ready to go!

**Comment by adam-mesos** [comment]
Thanks! Ship it! :rocket:

**Comment by elingg** [comment]
Looks good, that is better!

**Pull request merged**